### PR TITLE
DEXP-1585: Known issues and suggested workarounds for RBAC

### PR DESCRIPTION
**Pull request by asmita32675**
https://rackspace.atlassian.net/browse/DEXP-1585

### CODE REVIEW

**Review by ssalinas24 — CHANGES_REQUESTED** [review]

File: content/how-to/account-management/account-management-faq/index.md [hunk 1]
```diff
@@ -5,17 +5,17 @@ title: Account Management FAQ
 type: article
 created_date: '2019-05-16'
 created_by: Stephanie Fillmon
-last_modified_date: '2020-09-08'
-last_modified_by: Stephanie Fillmon
+last_modified_date: '2020-07-15'
```
Comment (ssalinas24) [inline content/how-to/account-management/account-management-faq/index.md:8]: Remember to update the date right before merge

File: content/how-to/account-management/account-management-faq/index.md [hunk 2]
```diff
@@ -30,13 +30,12 @@ Yes, RBAC is available to all Rackspace customers.
 
 {{< accordion title="When I add a user login to an existing contact, can I configure custom role assignments?" col="in" href="accordion19" >}}
 
-Yes, you can. For instructions, see [Use Role-Based Access Control
-(RBAC)](/support/how-to/managing-role-based-access-control-rbac).
+Yes, you can. For instructions, see [Use Role-Based Access Control (RBAC)](/support/how-to/managing-role-based-access-control-rbac).
```
Comment (ssalinas24) [inline content/how-to/account-management/account-management-faq/index.md:33]: Do any of your other PRs change the file name of the article found at this link? /support/how-to/managing-role-based-access-control-rbac

If so, then this link won't be able to find that article when user clicks it.
Just double check.

File: content/how-to/account-management/account-management-faq/index.md [hunk 3]
```diff
@@ -51,27 +50,22 @@ Per-billing and per-user usage are not offered at this time.
 
 {{< accordion title="Which users can create support tickets?" col="in" href="accordion22" >}}
 
-All users can create support tickets. However, only the account owner is
-updated on the ticket status.
+All users can create support tickets. However, only the account owner is updated on the ticket status.
```
Comment (ssalinas24) [inline content/how-to/account-management/account-management-faq/index.md:53]: This is incorrect. Didn't we change this in a different PR, where I typed out that admins and device admins on dedicated accounts can create tickets. Device admins can only create tickets that have a device on them. Observers can't create tickets.... I explained all that somewhere. Where?

**Review by ssalinas24 — CHANGES_REQUESTED** [review]

File: content/how-to/account-management/account-registration-code/index.md [hunk 4]
```diff
@@ -1,26 +1,21 @@
...
-Rackspace uses an _account registration code_ to verify new users on an
-account, much like two-factor authentication. If an account administrator
-adds you as a new user, an account registration code is generated for
-your new user account and you receive an email notification that
-walks you through how to log in to the
-[Cloud Control Panel](https://login.rackspace.com)
-for the first time. Your first login is the only time that you need the
-registration code, unless you forget your user name.
+Rackspace uses an _account registration code_ to verify new users on an account, much like two-factor authentication as shown in below image. 
+<img width="776" alt="accountregistrationcode1" src="/support/how-to/account-registration-code/accountregistrationcode1.png">
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/account-registration-code/index.md:13]: Added alt tag.

File: content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md [hunk 5]
```diff
@@ -1,67 +1,56 @@
...
 
-### Custom role
+### Custom Role
 
-The Custom role enables you to assign per-product access. After a user is
-assigned the Custom role, the account owner cannot change this role to a
-multiproduct role (All Products or No Product Access) by using the Cloud
-Control Panel.
+**Cloud Users**
+<img width="265" alt="customrole" src="/support/how-to/known-issues-and-suggested-workarounds-role-based-access-control-rbac/customrole.png">
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md:21]: Updated alt tag information.

File: content/how-to/account-management/managing-role-based-access-control-rbac/index.md [hunk 6]
```diff
@@ -5,102 +5,73 @@ title: Use Role Based Access Control (RBAC)
...
+Also, new users receive an email from Rackspace notifying them that they have been added to the account. They receive information on how to sign up for an account in the email.
 
-### Create new users
+### Create New Users
 
 To create a new user, use the following steps:
 
 1. Log in to the [Cloud Control Panel](https://login.rackspace.com/).
 2. In the upper-right corner of the control panel, click **Account > User Management.**
+<img width="946" alt="MultiAccountRBAC1" src="MultiAccountRBAC1.png">
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/managing-role-based-access-control-rbac/index.md:38]: Replaced image as suggested

File: content/how-to/account-management/managing-role-based-access-control-rbac/index.md [hunk 7]
```diff
@@ -5,102 +5,73 @@ title: Use Role Based Access Control (RBAC)
...
-account by an email they receive from Rackspace. In the email, they
-are given instructions about registering as a user of the account.
+Also, new users receive an email from Rackspace notifying them that they have been added to the account. They receive information on how to sign up for an account in the email.
 
-### Create new users
+### Create New Users
 
 To create a new user, use the following steps:
 
 1. Log in to the [Cloud Control Panel](https://login.rackspace.com/).
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/managing-role-based-access-control-rbac/index.md:36]: Done

File: content/how-to/account-management/account-management-faq/index.md (hunk 3 above)
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:53]: Yes, changes are requested in PLATEXP-1084

File: content/how-to/account-management/account-registration-code/index.md [hunk 8]
```diff
@@ -1,26 +1,21 @@
...
 permalink: account-registration-code
 audit_date: '2019-01-22'
-title: Account registration code
+title: Account Registration Code
 type: article
 created_date: '2019-02-08'
 created_by: Rackspace Community
-last_modified_date: '2019-02-08'
-last_modified_by: Kate Dougherty
+last_modified_date: '2022-08-01'
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/account-registration-code/index.md]: Date Change.

File: content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md [hunk 9]
```diff
@@ -1,67 +1,56 @@
...
 permalink: known-issues-and-suggested-workarounds-role-based-access-control-rbac
 audit_date: '2018-12-10'
-title: Known issues and suggested workarounds for RBAC
+title: Known Issues and Suggested Workarounds for RBAC
 type: article
 created_date: '2013-08-16'
 created_by: Renee Rendon
-last_modified_date: '2018-12-17'
-last_modified_by: Kate Dougherty
+last_modified_date: '2022-08-16'
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md]: Done

File: content/how-to/account-management/account-management-faq/index.md (hunk 1 above)
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:8]: Date change

File: content/how-to/account-management/account-management-faq/index.md (hunk 2 above)
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md]: Update the link [Use Role-Based Access Control (RBAC)](/support/how-to/managing-role-based-access-control-rbac) with the [Use Role-Based Access Control (RBAC)](/support/how-to/use-role-based-access-control-rbac).

File: content/how-to/account-management/managing-role-based-access-control-rbac/index.md [hunk 10]
```diff
@@ -5,102 +5,73 @@ title: Use Role Based Access Control (RBAC)
 type: article
 created_date: '2013-06-28'
 created_by: Renee Rendon
-last_modified_date: '2019-10-09'
-last_modified_by: Catherine Richardson
+last_modified_date: '2022-07-12'
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/managing-role-based-access-control-rbac/index.md]: Date change

File: content/how-to/account-management/account-management-faq/index.md [hunk 11]
```diff
@@ -5,17 +5,17 @@ title: Account Management FAQ
 type: article
 created_date: '2019-05-16'
 created_by: Stephanie Fillmon
-last_modified_date: '2020-09-08'
-last_modified_by: Stephanie Fillmon
+last_modified_date: '2023-02-21'
```
Comment (Rohi2305) [inline content/how-to/account-management/account-management-faq/index.md:8]: Change the date

File: content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md [hunk 12]
```diff
@@ -1,67 +1,58 @@
...
 permalink: known-issues-and-suggested-workarounds-role-based-access-control-rbac
 audit_date: '2018-12-10'
-title: Known issues and suggested workarounds for RBAC
+title: Known Issues and Suggested Workarounds for RBAC
 type: article
 created_date: '2013-08-16'
 created_by: Renee Rendon
-last_modified_date: '2018-12-17'
-last_modified_by: Kate Dougherty
+last_modified_date: '2023-02-21'
```
Comment (Rohi2305) [inline content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md:8]: change the date

File: content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md [hunk 13]
```diff
@@ -1,67 +1,58 @@
...
+**Cloud Users**
 
-#### Workaround
+   <img width="265" alt="Displays Cloud access permissions and types of services" src="/support/how-to/known-issues-and-suggested-workarounds-role-based-access-control-rbac/customrole.png">
 
-The account owner can simulate the Full Access role by
-assigning the user the Admin role for all available products. To simulate
-the Read-Only Access role, the account owner can assign the
-Observer role to the user for all available products.
+The Custom role enables you to assign per-product access. After a user is assigned the Custom role, the account owner cannot change this role to a multiproduct role (All Products or No Product Access) by using the Cloud Control Panel.
```
Comment (Rohi2305) [inline content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md:24]:
```suggestion
The Custom role enables you to assign access per-product . After a user is assigned the Custom role, the account owner cannot change this role to a multiproduct role (All Products or No Product Access).
```

File: content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md [hunk 14]
```diff
@@ -1,67 +1,58 @@
...
-update when new Rackspace products become RBAC-enabled. The account owner must
-update Custom roles to include the new products.
+**Dedicated Users**
 
-You can also change the Custom role to a multiproduct role by contacting
-Support at 1 800 961 4454.
+   <img width="883" alt="Assign required permissions to user" src="/support/how-to/known-issues-and-suggested-workarounds-role-based-access-control-rbac/customrole_dedicated.png">
 
-### Multiple roles for one user
+For dedicated users there are two types of permissions such as direct and effective permissions. By selecting None, View, Edit, or Admin under the direct permission column, the user, user group, or group membership receives the required permissions.
```
Comment (Rohi2305) [inline content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md:30]:
```suggestion
For dedicated users there are two types of permissions such as direct and effective permissions. The User, User Group, or Group Membership can receive the required permissions by selecting None, View, Edit, or Admin under the direct permission column.
```

File: content/how-to/account-management/account-management-faq/index.md (hunk 11 above)
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:8]: Done

File: content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md (hunk 13 above)
Comment (asmita32675) [reply content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md:24]: Done

File: content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md (hunk 14 above)
Comment (asmita32675) [reply content/how-to/account-management/known-issues-and-suggested-workarounds-role-based-access-control-rbac/index.md:30]: Done

**Review by ssalinas24 — APPROVED** [review]

**Pull request merged**